docs/src/sections/NavbarSection.js

```diff
@@ -41,12 +41,14 @@ export default function NavbarSection() {
 
       <h3><Anchor id="navbars-mobile-friendly">Responsive Navbars</Anchor></h3>
       <p>
-        To have a mobile friendly Navbar, Add a <code>Navbar.Toggle</code> to your Header and wrap your
-        Navs in a <code>Navbar.Collapse</code> component. The <code>Navbar</code> will automatically wire
-        the toggle and collapse together!
+        To have a mobile-friendly Navbar, add a <code>Navbar.Toggle</code> to your Header and wrap your
+        Navs in a <code>Navbar.Collapse</code> component. The <code>Navbar</code> will automatically combine
```

Comment:
Sorry if the wording had become just as ambiguous as it was before, I had hoped to make it a bit clearer in the rewrite but I wasn't super familiar with the exact nuances of how this worked. I'm glad we were able to talk and clear it up though 😅